Integrations/KennaV2/KennaV2.py

```diff
@@ -0,0 +1,480 @@
+from json import JSONDecodeError
+from typing import List, Tuple, Dict, Any, Optional, Callable
+
+import urllib3
+import demistomock as demisto
+from CommonServerPython import *
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+class Client(BaseClient):
+    def __init__(self, base_url: str, api_key: str, verify: bool, proxy: bool):
+        header = {
+            'X-Risk-Token': api_key,
+            'Content-Type': 'application/json',
+            'Accept': 'application/json'
+        }
+        super().__init__(base_url=base_url, verify=verify, proxy=proxy, headers=header)
+
+    def http_request(self, message: str, suffix: str, params: Optional[dict] = None,
+                     data: Optional[dict] = None):
+        return super()._http_request(message, suffix, params=params, json_data=data)
+
+
+def test_module(client: Client, *_):
+    """
+    Performs basic get request to get item samples12
+    """
+    res_vulnerabilities = client._http_request('GET', '/vulnerabilities')
+    res_assets = client._http_request('GET', '/assets')
+
+    if isinstance(res_vulnerabilities.get('vulnerabilities'), list) and isinstance(res_assets.get('assets'), list):
+        return 'ok', None, None
+    else:
+        raise Exception('Error occurred while trying to query the api.')
+
+
+# ----------------------------------------------- Auxiliary functions -------------------------------------------------
+
+
+def connect_api(client: Client, message: str, suffix: str, params: Optional[dict] = None,
+                data: Optional[dict] = None) -> Dict[str, Any]:
+    """Connects to api and Returns response.
+    Args:
+        client:  BaseClient which connects to api
+        message: The HTTP message, for example: GET, POST, and so on
+        suffix :The API endpoint.
+        params: URL parameters to specify the query.
+        data:The data to send in a specific request.
+    Returns:
+        response from the api.
+    """
+    raw_response = client.http_request(message, suffix, params=params, data=data)
+    # Check if response contains errors
+    if raw_response.get('errors'):
+        raise DemistoException(raw_response.get('errors'))
+    elif raw_response.get('error'):
+        raise DemistoException(raw_response.get('error'))
+    response_list = raw_response
+    return response_list
+
+
+def create_dict(raw_data: List[Dict[str, Any]], wanted_keys: List[Any], actual_keys: List[Any]) -> List[Dict[str, Any]]:
```

Comment:
please change to a more indicative name, e.g:
parse_response/outputs, create_context_dict

Reply:
also, aux functions are usually above the client class